recipe_batches/recipe_batches.py

```diff
@@ -2,9 +2,23 @@
 
 import math
 
+# how many batches of a particular recipe can we make 
+  # ingredients // recipe  for each value
+  # min of all of the values would be how many batches there are
+#recipe key values list > than ingredients return 0
+#recipe and ingredients have the same keys
+#ingredients.get(key, 0) would just give me a 0 if key missing
 def recipe_batches(recipe, ingredients):
-  pass 
-
+  batches = []
+  for item in recipe: #item in list of values in recipe
+    if item not in ingredients:
+      batches.append(0) # appending 0 for value of items not there
+    elif recipe[item] > ingredients[item]:
+      batches.append(0) 
+    else:
+      batches.append(ingredients[item]//recipe[item]) 
```

Comment:
Great use of appending the ratio directly. You don't have to mutate the dictionary by doing this.